Sources/SentryDistribution/Utils/Date+Parse.swift

```diff
@@ -0,0 +1,9 @@
+import Foundation
+
+extension Date {
+  static func fromString(_ input: String) -> Date? {
+    let formatter = ISO8601DateFormatter()
+    formatter.formatOptions = [ .withInternetDateTime, .withFractionalSeconds ]
+    return formatter.date(from: input)
+  }
```

Comment:
`l`: IIRC it's best practice to reuse date formatters. Probably not possible if this stays an extension of Date.

Rough idea:
```suggestion
  private static var iso8601Formatter: ISO8601DateFormatter = {
    let formatter = ISO8601DateFormatter()
    formatter.formatOptions = [ .withInternetDateTime, .withFractionalSeconds ]
    return formatter
  }()
  
  static func fromString(_ input: String) -> Date? {
    iso8601Formatter.date(from: input)
  }
```